### DISCUSSION TITLE
Enum

### DISCUSSION THREAD

**Discussion by imtiaz70** [nehal2210/Blockchain-Engineer-Course#7] [Q&A] [answered]
when we can access enum directly **Status.Pending** , Why we have to make its object **status.pending** ?
can we call Enum as Fixed Array ?

**Comment by nehal2210** [comment, marked as answer]
@imtiaz70 , you can use it directly, but when you have to store it, you must create an enum Object to store this value.
ForExample : 
if you have a scenario to check the condition you can access the enum directly.
` require(Status.Pending == Something)`

please make this comment as an "answer" if this is satisfactory for you or you can ask further about it.

**Reply by imtiaz70** [reply]
Thank You  @nehal2210
